CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.9.1 November 7, 2022
 
+## Bug fixes 🐞
+* Handle OOB when getting DEM Data. ([#1808](https://github.com/mapbox/mapbox-maps-android/pull/1808))
```

Comment:
What is `OOB` ?

Reply:
Ah, `out of bounds`, checked in the gl repo. Not sure it's that clear, maybe just `Fix crash getting DEM Data.` instead ?

Reply:
Out of bounds

Reply:
we have same wordings in 10.10-beta.1 [here](https://github.com/mapbox/mapbox-maps-android/pull/1791/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15), should we then change to both places ?

Reply:
Ah, no don't think it's worth it, lets stick to the old one